packages/algob/sample-project/algob.config.js

```diff
@@ -5,49 +5,42 @@
 // NOTE: to be able to execute transactions, you need to use an active account with
 // a sufficient ALGO balance.
 
+/**
+  Check our [algob-config.md](/docs/algob-config.md) documentation for more ways how to
+  load a private keys:
+  + using mnemonic
+  + using binary secret key
+  + using KMD daemon
+  + loading from a file
+  + loading from an environment variable
+  + ...
+*/
+
 // ## ACCOUNTS USING mnemonic ##
-// const { mkAccounts } = require("algob");
-// let accounts = mkAccounts([{
-//   addr: "KFMPC5QWM3SC54X7UWUW6OSDOIT3H3YA5UOCUAE2ABERXYSKZS5Q3X5IZY",
-//   mnemonic: "call boy rubber fashion arch day capable one sweet skate outside purse six early learn tuition eagle love breeze pizza loud today popular able divide"
-// }]);
-//
-// To see your algob account and extract menemonic run:
-// 	goal -d $(ALGORAND_DATA) account list
-// 	goal -d $(ALGORAND_DATA) account export -a <account address>
+const { mkAccounts } = require("algob");
+let accounts = mkAccounts([{
```

Comment:
this config also needs to have async loading `module.exports = (async function() {` else it fails
giving this error.
![Uploading ss12.png…]()

Reply:
i think this needs to be updated for all example projects, should i update it ?

Reply:
Thanks for catching. I want to avoid `async` in config. We shouldn't use it here.

Reply:
The problem was in the exports, I forgot to add a `networks` layer.